cart.php

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>Your Shopping Cart</title>
+<meta charset="utf-8" />
+<link rel="stylesheet" href="style.css" media="screen" type="text/css" />
+<script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jquery/2.0.3/jquery.min.js"></script>
+<script type="text/javascript" src="jquery.store.js"></script>
+</head>
+
+<body>
+
+<div id="site">
+	<div id="content">
+		<h1>Your Shopping Cart</h1>
+			<table class="shopping-cart">
+			  <thead>
+				<tr>
+					<th scope="col">Item Name</th>
+					<th scope="col">Price</th>
+				</tr>
```

Comment:
please add the comment that the PHP code started which saves time for other developers to find it easily